api/python/stat_all.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Place Statistics - All
+nav_order: 14
+parent: Python
+grand_parent: API
+---
+
+# Get a Collection of Statistical Data for Multitple Places
+
+## `datacommons.get_stat_all(places, stat_vars)`
+
+Returns a nested `dict` of all time series for [`places`](https://datacommons.org/browser/Place) and [`stat_vars`](https://datacommons.org/browser/StatisticalVariable).
+
+**Arguments**
+
+- `places (Iterable of str)`: The `dcid`s of the [`Place`](https://datacommons.org/browser/Place) to query for.
+
+- `stats_vars (Iterable of str)`: The `dcid`s of the
+  [`StatisticalVariable`](https://datacommons.org/browser/StatisticalVariable).
+
+**Returns**
+
+A nested `dict` mapping Places to StatisticalVariables and all available
```

Comment:
Add backticks to `Place`s and `StatisticalVariable`s to keep things consistent in this page?

Reply:
done